### PR TITLE
methods #forgetAllBackward & #forgetAllForward

### PR DESCRIPTION
**Pull request by mightyiam**
(including tests)

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -44,5 +44,8 @@ History.prototype.lengthForward = require('./prototype/length-forward')
 History.prototype.backward = require('./prototype/backward')
 History.prototype.forward = require('./prototype/forward')
 History.prototype.move = require('./prototype/move')
+History.prototype.forgetAllInDirection = require('./prototype/forget-all-in-direction')
```
Comment (winhamwr) [inline lib/index.js:47]: This doesn't seem like a useful external API. Why would someone prefer passing in a magic string over calling the appropriate method directly?

Comment (winhamwr) [inline lib/index.js:47]: It seems useful as an internal method, though, just not worth complicating the API that's exposed to library users.

Comment (mightyiam) [inline lib/index.js:47]: [It is private](https://github.com/PolicyStat/object-history/pull/9/files#diff-5f8ebb50eb0dbedd1ebad3dc174692e4R7). I did not follow the convention of prefixing `_` or any means of hiding the method.

The only thing I did is document it as private. Which means that Verb excludes it from the documentation.

Comment (mightyiam) [inline lib/index.js:47]: This project [has documentation](https://github.com/PolicyStat/object-history).

The API part is from the code comments.

This new method, as I wrote, isn't there.

**Pull request merged**